src/Slingshot.vala

```diff
@@ -87,6 +106,53 @@ public class Slingshot.Slingshot : Wingpanel.Indicator {
     public override void closed () {
         // TODO: Do we need to do anyhting here?
     }
+
+    private void update_tooltip () {
+        if (keybinding_settings == null || indicator_grid == null) {
+            return;
+        }
+
+        string[] accels = keybinding_settings.get_strv ("panel-main-menu");
+        if (accels.length > 0) {
+            string shortcut = accel_to_string (accels[0]);
+            indicator_grid.tooltip_text = (_("Open and search apps (%s)").printf (shortcut));
+        }
+    }
+
+    private static string accel_to_string (string accel) {
+        string[] keys = parse_accelerator (accel);
+        return string.joinv (_(" + "), keys);
+    }
+
+    private static string[] parse_accelerator (string accel) {
+        uint accel_key;
+        Gdk.ModifierType accel_mods;
+        Gtk.accelerator_parse (accel, out accel_key, out accel_mods);
+
+        string[] arr = {};
+        if (Gdk.ModifierType.SUPER_MASK in accel_mods) {
+            arr += _("⌘");
+        }
+
+        if (Gdk.ModifierType.SHIFT_MASK in accel_mods) {
+            arr += _("Shift");
+        }
+
+        if (Gdk.ModifierType.CONTROL_MASK in accel_mods) {
+            arr += _("Ctrl");
+        }
+
+        if (Gdk.ModifierType.MOD1_MASK in accel_mods) {
+            arr += _("Alt");
+        }
+
+        string? key = Gdk.keyval_name (accel_key);
+        if (key != null) {
+            arr += key;
```

Comment:
Can we capitalize the first letter here? So `l` becomes "L" and `space` becomes "Space"